tests/debugging_primitives_test.py

```diff
@@ -214,9 +214,38 @@ def f(x):
 
     with capture_stdout() as output:
       x = jnp.array(1., jnp.float32)
-      jax.jvp(f, (x,), (x,))  # should print out cos(1.)
+      jax.jvp(f, (x,), (x,))
       jax.effects_barrier()
-    self.assertEqual(output(), "x_tangent: 0.5403022766113281\n")
+    self.assertEqual(output(), f"x_tangent: {jnp.cos(1.)}\n")
+
+  @unittest.skip("doesn't work yet!")  # TODO(mattjj,sharadmv)
+  def test_debug_print_in_custom_jvp_linearize(self):
+
+    @jax.custom_jvp
+    def print_tangent(x):
+      return x
+
+    @print_tangent.defjvp
+    def _(primals, tangents):
+      (x,), (t,) = primals, tangents
+      debug_print("x_tangent: {}", t)
+      return x, t
+
+    def f(x):
+      x = jnp.sin(x)
+      x = print_tangent(x)
+      return x
+
+    with capture_stdout() as output:
+      x = jnp.array(1., jnp.float32)
+      y, f_lin = jax.linearize(f, x)
+      jax.effects_barrier()
+    self.assertEqual(output(), "")
```

Comment:
This passes right?

Reply:
Actually no, nothing is printed. Try it yourself!